core/socket/socket.go

```diff
@@ -1,26 +1,25 @@
-package core
+package socket
```

Comment:
This got split out to its own package as part of clipping dependencies from the CLI to various buildkit/containerd packages